package.json

```diff
@@ -69,6 +69,14 @@
       "es2015"
     ]
   },
+  "nyc": {
+    "extension": [
+      ".raw"
```

Comment:
`lib/hook.template.raw` wouldn't be included in the coverage report otherwise (by default, `nyc` only includes files with `.js` extension, outside a `./test` folder).